### ISSUE TITLE
[$15][MSFT-223] Sample challenge page: Colour contrast issues.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2882]
## **Note: Summary of fixes needs are here https://github.com/topcoder-platform/community-app/issues/2882#issuecomment-516881272**

**Steps to Reproduce**
1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347

**Expected Result**
The colour contrast should comply as per the WCAG minimum contrast requirements.

**Actual Result**
The colour contrast between the background and foreground of the elements doesn't comply with WCAG minimum contrast requirements.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2038

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095980 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095980 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu is it possible to elaborate a bit more? where color needs to be changed? hex values if given?

**Comment by veshu** [comment]
@PrakashDurlabhji please check the attachment. it is listed on many places. 

And sorry for inconvenience but please don't work on this, I will like to get fix this at last. Thanks!

**Comment by Dara-K** [comment]
Please use #006DEA as the link color in all the places.
For the red text: #EA1900
Register button: #006AD7  
All  gray text on timeline bar: #D5D5D5
"Prizes": #6B6B6B
Login button: let’s have a white button, border color #C3C3C8 and text color #47474F
The Submit button is in disabled state, if we would make it darker it would look active.
The tag should be updates should be part of a larger task of updating all tags for all tracks.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - it has been assigned to veshu.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by veshu** [comment]
Things to fix for this issue 
1. All gray text on timeline bar: #D5D5D5
2. Prizes section "Prizes": #6B6B6B, 1st, 2nd etc $6b6b6b
3. Track tag contrasts for all tracks(Color to be provided from @Dara-K )
4. Active blue links 006DEAd (Detail, Registrants, Submissions, Winners etc

Red section is added at run time so no need to fix.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by Gauravseta** [comment]
I hope it is with me now

**Comment by veshu** [comment]
@Gauravseta Yes please go on. Thanks!

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096757 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@veshu as mentioned in your summary :

1. completed
2. Completed
3. waiting for colors from dara-k (data science, development, design all shades)
4. Completed

so only 3rd point left, waiting for @Dara-K comment.

**Comment by veshu** [comment]
@rashmi73 Please see #3184 for the 3 point, we are moving this to deployment.
Thanks!

**Comment by drasticdpk** [comment]
@rashmi73 , Please update below color as suggested. 
4. Active blue links 006DEAd (Detail, Registrants, Submissions, Winners etc
![image](https://user-images.githubusercontent.com/25297859/62310266-d86c6a80-b4a8-11e9-9f4b-ed1721082b4b.png)

**Comment by rashmi73** [comment]
@drasticdpk yes updating it, but i guess it is just #006dea and not "#006dead"

**Comment by rashmi73** [comment]
@drasticdpk kindly find 2 PR raised to satisfy your above commented feedback.

https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/22
https://github.com/topcoder-platform/community-app/pull/3190

**Comment by veshu** [comment]
> @drasticdpk yes updating it, but i guess it is just #006dea and not "#006dead"
> 
> @veshu I guess in color palette such color is not present under any variable in topcoder-react-ui kit so shall I make a new variable? what should be the name of variable?

Sorry this is not required, after Darak suggested here, she has updated the color palette which is 006ad7

**Comment by sushilshinde** [comment]
@veshu It is ready for QA and there are no PR associated with it, please explain

**Comment by rashmi73** [comment]
@sushilshinde kindly check comment above and link for it is :
https://github.com/topcoder-platform/community-app/issues/2882#issuecomment-517068536

actually 4 tasks were originally in this issue, but 3rd one was confusing hence for other 3 points PR was rasied  #3183 and merged, and for 3rd point colors were left out by dara-k
so was waiting for it.

so for 3rd task new issue was raised and I raised PR for that alone but then requirement for 3rd task was not required and hence my separate PR was closed and you went through that separate PR which u seen as closed.

whereas original PR was merged hence Ready for QA label was present.

**Comment by veshu** [comment]
@sushilshinde Yes as rashmi said I have separated the issue which causes the failure of the is issue. so it is ready for testing. 

We have separate PR and issue for failing reason. Thanks!

**Comment by lakshmiathreya** [comment]
Please clarify the overall goal of this Issue. I continue to see several contrast issues.

**Comment by tosha5252** [comment]
@veshu I'm reading the comments and from what I understand, for this issue, points 1,2 and 4 need to be reverified and that is all that is required for it to be closed...because point 3 was seperated and is its own issue...is that correct?

**Comment by crazyk07** [comment]
@tosha5252  Yes, lets go with this.  If the three issues are solved (or all issues are already solved) lets go with this as complete.  If point "3" is not solved, lets raise a separate ticket, but close this one out.

cc: @lakshmiathreya

**Comment by lakshmiathreya** [comment]
noted. @tosha5252 - lets proceed as per @crazyk07 recommendation on this one.

**Comment by tosha5252** [comment]
noted.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@tosha5252 @veshu wont i be paid for it? no paid label?

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096757<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment failed: [400]: Failed to update challenge. Detail: javax.ejb.EJBTransactionRolledbackException<br/><br/>```This is an automated message for crazyk via Topcoder X```
